deploy.sh

```diff
@@ -2,41 +2,17 @@
 
 set -euxfo pipefail;
 
-declare -r LATEST_RACKET_VERSION="7.3";
+source "_common.sh";
 
-docker image tag "jackfirth/racket:${LATEST_RACKET_VERSION}" "jackfirth/racket:latest";
+declare -r LATEST_RACKET_VERSION="7.4";
 
-push () {
-  declare -r version="${1}";
-  docker image push "jackfirth/racket:${version}";
-};
+docker image tag "${USERNAME}/racket:${LATEST_RACKET_VERSION}" "${USERNAME}/racket:latest";
```

Comment:
This relies on the `USERNAME` variable being defined in `_common.sh`, correct?

Reply:
Yes, that's right. Each of the scripts sets a bunch of flags at the very beginning (the `set -euxo pipefail` bit). One of those flags, `-u`, tells the interpreter to exit immediately if an undeclared variable is used. So, if we had forgotten to include `_common.sh`, then the script wouldn't trample over our environment and would just exit with an error instead.